scaladoc/src/dotty/tools/scaladoc/tasty/comments/Comments.scala

```diff
@@ -252,29 +251,44 @@ class MarkdownCommentParser(repr: Repr, snippetChecker: SnippetChecker)(using dc
     val nodes = root.getDescendants().asScala.collect {
       case fcb: mda.FencedCodeBlock => fcb
     }.toList
-    if !nodes.isEmpty then {
```

Comment:
I think this condition should be left because majority of members don't have any snippet. With this `if` we don't collect all that info needed to run snippet if there isn't any to run.

Reply:
aye aye